traveler_ui_kit/src/main/AndroidManifest.xml

```diff
@@ -12,13 +12,19 @@
                 android:name=".activities.BookingItemDetailsActivity"
                 android:launchMode="singleTop">
         </activity>
+        <activity
+                android:name=".activities.ParkingItemDetailsActivity"
+                android:launchMode="singleTop">
```

Comment:
why singleTop?

Reply:
Was intended to mirror BookingItemDetailsActivity's singletop, if ParkingItemDetailsActivity can conceivably be started from other places